c/tskit/genotypes.c

```diff
@@ -427,11 +515,20 @@ tsk_vargen_update_site(tsk_vargen_t *self)
             update_genotypes = tsk_vargen_update_genotypes_i8_traversal;
         }
     }
-
-    /* Ancestral state is always allele 0 */
-    var->alleles[0] = site->ancestral_state;
-    var->allele_lengths[0] = site->ancestral_state_length;
-    var->num_alleles = 1;
+    if (self->fixed_alleles) {
```

Comment:
"fixed" has an unfortunate double connotation here - at first I was thinking about "fixed" as in "everyone has it", which is the same thing as "ancestral". Other option: `has_allele_map`?

Reply:
Good point, we should change this. I'll have a think about it.

Reply:
Also perhaps change the PR name, so it's clearer when looking back at the git history?